sdk/eventhub/event-hubs/src/eventHubSender.ts

```diff
@@ -742,19 +743,20 @@ export class EventHubSender extends LinkEntity {
         }
       });
 
-    const maxRetries = options.retryOptions && options.retryOptions.maxRetries;
+    const maxRetries = retryOptions.maxRetries;
     const delayInSeconds =
-      options.retryOptions &&
-      options.retryOptions.retryInterval &&
-      options.retryOptions.retryInterval >= 0
-        ? options.retryOptions.retryInterval / 1000
-        : Constants.defaultDelayBetweenOperationRetriesInSeconds;
+      typeof retryOptions.retryInterval === "number"
+        ? retryOptions.retryInterval / 1000
+        : undefined;
```

Comment:
The above 2 constants are not needed. They can be inlined just like we do in other places where we create the `RetryConfig`

Reply:
Then as part of #4401, the inline will get simplified too